test/integration/scale/scale_test.go

```diff
@@ -55,12 +55,11 @@ func TestMain(m *testing.M) {
 func TestScaleSubresources(t *testing.T) {
 	clientSet, tearDown := setupWithOptions(t, nil, []string{
 		"--runtime-config",
-		// TODO(liggitt): remove these once apps/v1beta1, apps/v1beta2, and extensions/v1beta1 can no longer be served
-		"api/all=true,extensions/v1beta1/deployments=true,extensions/v1beta1/replicationcontrollers=true,extensions/v1beta1/replicasets=true",
+		"api/all=true",
 	})
 	defer tearDown()
 
-	resourceLists, err := clientSet.Discovery().ServerResources()
+	_, resourceLists, err := clientSet.Discovery().ServerGroupsAndResources()
```

Comment:
For such a big upgrade, please don't make unnecessary code changes next time

Reply:
ServerResources is not available any more.

Reply:
I am still seeing ServerResources() in client-go/discovery/discovery_client.go